.snyk

```diff
@@ -4,6 +4,6 @@ ignore:
   SNYK-DEBIAN10-GNUTLS28-609778:
     - '*':
         reason: Ticket to change base image raised.
-        expires: 2020-10-13T00:00:00.000Z
+        expires: 2020-10-30T00:00:00.000Z
```

Comment:
Ah that's one thing I forgot to check when I did the image update is our Snyk policy.  I think we can remove this now. I've got another PR open so leave it to me, we can merge this as is.

Reply:
OK